app/Http/Controllers/Payment/CurrencyController.php

```diff
@@ -103,11 +103,11 @@ public function getButtonColor($id)
         }
         $currency = Currency::where('id', $id)->pluck('dashboard_currency')->first();
         if ($currency == 1) {
-            return '<a href='.url('dashboard-currency/'.$id)." 
+            return '<a href='.action('Payment\CurrencyController@setDashboardCurrency', [$id])." 
```

Comment:
what difference will it make in href link? In fresh installation not working throwing this response
```
{"draw":1,"recordsTotal":161,"recordsFiltered":0,"data":[],"error":"Exception Message:\n\nAction App\\Http\\Controllers\\Payment\\CurrencyController@setDashboardCurrency not defined."}
```

Reply:
Fixed